pyuntl/untl_structure.py

```diff
@@ -457,6 +458,48 @@ def generate_form_data(self, **kwargs):
         # Create the form object.
         return FormGenerator(**kwargs)
 
+    def make_hidden(self):
+        """Make an unhidden UNTL element into a hidden element."""
+        meta_hidden = False
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                meta_hidden = True
+                # Make the element hidden.
+                if element.content == 'False':
+                    element.content = 'True'
```

Comment:
A couple thoughts here that apply similarly to `make_hidden` and `make_unhidden`:
1. If we have gone into the `if element.tag == 'meta' and element.qualifier == 'hidden'` block, than rather than keep looping after we finish what we are doing here, we should be done and thus could have a `return None` to get out of the method. This would also allow getting rid of the `meta_hidden` element because if we cycle through all the elements and haven't returned, we know we need to create the element.
2. If you don't want to do the above for some specified reason and prefer to keep `meta_hidden`, another thing we might consider is `break` out of the loop after we have found `element.tag == 'meta' and element.qualifier == 'hidden'` to save on iterating over extra children after we found what we are looking for.